aries_cloudagent/core/conductor.py

```diff
@@ -304,7 +308,7 @@ async def start(self) -> None:
                 LOGGER.exception("Error creating invitation")
 
         # Accept mediation invitation if specified
-        mediation_invitation: str = context.settings.get("mediation.invite")
+        mediation_invitation: str = context.settings.get("mediation.invite", None)
```

Comment:
get called on a dictionary returns `None` by default, no need to be explicit.  I prefer without None.